cluster/examples/kubernetes/ceph/upgrade-from-v1.0-apply.yaml

```diff
@@ -98,6 +98,12 @@ rules:
   - create
   - update
   - delete
+- apiGroups:
```

Comment:
Patch releases don't typically update the RBAC so i'd be concerned about backporting this. Are you needing to backport this? If not, no need to update this file. If so, we should modify the upgrade guide with steps for this in the patch release section.

Reply:
removed RBAC